packages/retail-ui-extensions/src/extension-api/toast-api/toast-api.ts

```diff
@@ -0,0 +1,15 @@
+export interface ToastApiContent {
+  /**
+   * Show a toast.
+   * @param content The text content to display.
+   * @param duration The duration of the toast in ms. Defaults to 5000.
+   */
+  show: (content: string, duration?: number) => void;
```

Comment:
Do you think there is value giving partners to ability to `dismiss` their toast?

Reply:
I didn't really think so, since guidance on Toasts suggest to only use them for less important things. Between duration and all toasts having an "X" button, I was hard pressed to think of a case where we'd need to `dismiss`. Your thoughts?